src/subscriptions/reaperLambda/subscriptionReaper.ts

```diff
@@ -26,10 +30,12 @@ const reaperHandler = async (dbService: DynamoDbDataService, dbServiceWithTenanc
             })
             .map(async (subscription) => {
                 // delete the subscription as it has reached its end time
+                // multi-tenant deployments have the .id as {tenantid}|{id}
+                // eslint-disable-next-line no-underscore-dangle
+                const id = multiTenancyEnabled ? subscription._id : subscription.id;
```

Comment:
This fix looks good. Ideally you could have done it in a separate PR and have it merged right away. The integ tests take much longer to write/review